pandas/tests/io/test_clipboard.py

```diff
@@ -391,9 +392,9 @@ def test_round_trip_valid_encodings(self, enc, df):
 
     @pytest.mark.parametrize("data", ["\U0001f44d...", "Ωœ∑´...", "abcd..."])
     @pytest.mark.xfail(
-        reason="Flaky test in multi-process CI environment: GH 44584",
```

Comment:
Could you add the `@pytest.mark.single_cpu` marker on this test to replace this as well?

Reply:
Like this ? https://github.com/pandas-dev/pandas/pull/48520/commits/932dad55d66db2a523ed109eb1ae880d44bffe60

Is there an easy way to squash my commits such that there is only one left for the merge?

Reply:
Sure (thought typically we put these marks as the first decorator)

> Is there an easy way to squash my commits such that there is only one left for the merge?

No need to worry about it.  We squash all commits on merge.

Reply:
Thank you.

> (thought typically we put these marks as the first decorator)

Oh! Sorry! 
It should be good now : https://github.com/pandas-dev/pandas/pull/48520/commits/25b5527934f04d5e97341b662e4bc9d71c615424